lms/templates/main.html

```diff
@@ -140,16 +120,7 @@
     </div>
 
     % if not disable_footer:
-        <%block name="footer">
-          ## Can be overridden by child templates wanting to hide the footer.
-          % if theme_enabled() and not is_microsite():
-            <%include file="theme-footer.html" />
-          % elif settings.FEATURES.get('IS_EDX_DOMAIN', False) and not is_microsite():
-              <%include file="footer-edx-v3.html" />
-          % else:
-            <%include file="${microsite.get_template_path('footer.html')}" />
-          % endif
-        </%block>
+        <%include file="themable-footer.html" />
```

Comment:
`header.html` is very small and replaceable.  `footer.html` has more in it, including copyright and logos that we'd like people to leave in place.  We can look at this asymmetry and try to come up with a better solution.